### ISSUE TITLE
Failed to create post

### ISSUE THREAD

**Issue by Megachip** [nextcloud/social#1271]
**Describe the bug**

**To Reproduce**
1. Go to 'social'
2. Create a post 
3. tap on post

**Expected behavior**
A Post will be created 

**Client details:**
 - OS: macOS 11.3.1 (20E241)
 - Browser Safari
 - Version 14.1 (16611.1.21.161.6)
 - Device: MBP

<details>
<summary>Server details</summary>

**Social app version:** 0.4.2

**Operating system**: Linux 4.9.0-12-amd64 x86_64

**Web server:** nginx 

**Database:** mysql 5.7.33

**PHP version:** 7.3.28

**Nextcloud version:** 21.0.2

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
[no app in context] Warning: 500 - {"status":-1,"exception":"Doctrine\\DBAL\\Exception\\DriverException","message":"An exception occurred while executing a query: Unknown parameter type, 64 given."}

POST /index.php/apps/social/api/v1/post
from 213.61.89.122 by admin at 2021-05-31T12:00:16+00:00
```
</details>

**Comment by Megachip** [comment]
Maybe duplicate of #1208 ... didn't find this via search :/

**Comment by Megachip** [comment]
fixed by commit ce462c7
